assets/src/edit-story/panels/elements/getCommonValue.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
This file should like in a utils dir. IMO

Reply:
Fixed in d68854468.

Reply:
(and yes, I did think it was kind of out of place, but thought "ah, what the heck" - so thanks for keeping me on my toes 😜)